byoqm/models/code_climate.py

```diff
@@ -80,7 +81,27 @@ def method_count(self):
         return count
 
     def method_length(self):
-        pass
+        py_files = self.src_root.glob("**/*.py")
+        count = 0
+        for file in py_files:
+            with open(file) as f:
+                tree = self._parser.parse(bytes(f.read(), "utf8"))
+                children = tree.root_node.children
+                for child in children:
+                    if child.type != "function_definition":
+                        continue
+                    query = self._py_language.query(
+                        """
+                    (function_definition
+                        name: (identifier) @function.def)
+                    (call 
+                        function: (identifier) @function.call)"""
+                    )
+                    captures = query.captures(child)
```

Comment:
We should check if CodeClimate measures lines of code strictly or equates it to number of statements.